gen/definitions/policyassignments.yaml

```diff
@@ -0,0 +1,66 @@
+---
+name: Policy Assignments
+rest_endpoint: /api/fmc_config/v1/domain/{DOMAIN_UUID}/assignment/policyassignments
+no_delete: true
+no_data_source: true
+data_source_name_query: true
+doc_category: Policy Assignments
+attributes:
+  - model_name: type
+    type: String
+    value: "PolicyAssignment"
+  - model_name: name
+    type: String
+    description: User-created name of the resource.
+    mandatory: false
+    example: AccessPolicyName
+    data_path: ["policy"]
+    exclude_example: true
+    exclude_test: true    
+  - model_name: type
+    type: String
+    mandatory: false
+    example: AccessPolicy
+    data_path: ["policy"]
+    exclude_example: true
+    exclude_test: true    
```

Comment:
Description missing. This is the type of the policy to be assigned. Also, this duplicates the already existing model_name

Reply:
Description added;
model_names 'type' are different in generated code:
checks = append(checks, resource.TestCheckResourceAttr("data.fmc_policy_assignments.test", **"type"**, "AccessPolicy"))
and
checks = append(checks, resource.TestCheckResourceAttr("data.fmc_policy_assignments.test", **"targets.0.type"**, "DeviceGroup"))